docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/06-Zeit-Now-and-Google-Cloud-SQL.md

```diff
@@ -0,0 +1,158 @@
+---
+alias: joofei3ahd
```

Comment:
sorry I didn't mention this, but the `alias` needs to be unique among all tutorial pages. this one is already taken by the Fargate tutorial, you can use this one instead `mohj5eiwot`